website/content/docs/setup/quickstart/index.md

```diff
@@ -4,5 +4,53 @@ weight: 10
 ---
 
 This guide lets you quickly evaluate {{< name >}}. For a custom setup follow the [Installation instructions](/docs/setup/install).
- 
 
+## Before you begin
+Ensure that you have the following:
+- `kubectl` with access to a Kubernetes cluster (this guide was tested with kind v0.10.0 and OpenShift 4.3)
+- `git`
+- `make`
+
+## About this sample
+By the end of this guide you will have the following installed to your Kubernetes cluster:
+- The control plane of {{< name >}}
+- Hashicorp Vault and connector as the credentials manager 
+- ODPi Egeria lab and connectors as the data catalog
+- Open Policy Agent (OPA) and connector as the policy manager.
+- Arrow-Flight data access module for reading Parquet and CSV datasets
+
+## Prepare for installing {{< name >}}
+
+1.  Obtain a local copy of {{< name >}} repository
+    ```bash
+    git clone https://github.com/IBM/the-mesh-for-data.git
```

Comment:
Use shortcodes

Reply:
what do you mean?

Reply:
https://ibm.github.io/the-mesh-for-data/contribute/documentation/write/syntax/#add-links-to-content-on-github